test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/project.json

```diff
@@ -23,6 +23,7 @@
     },
     "net451": {
       "frameworkAssemblies": {
+        "System.DirectoryServices": "",
```

Comment:
What APIs are you using that you weren't using before (or did I just miss that).